.ci/test-project/spack_configs/linux_ubuntu_22/spack.yaml

```diff
@@ -0,0 +1,18 @@
+# Spack env file for ubuntu 22
+spack:
+  packages:
+    cmake:
+      externals:
+      - spec: cmake@3.14.5
```

Comment:
Can you verify this cmake version?

Reply:
it seems to find it fine here: https://github.com/LLNL/uberenv/actions/runs/3953297149/jobs/6769414659#step:4:114

`[+] /usr (external cmake-3.14.5-ne5mma6cz46xq6alhyop7qjelao6b55r)`

Reply:
I am pretty sure it just takes what version you said on line 6 and doesn't actually verify the version number.

You either have to add a `cmake --version` to the CI script and look at the output or download the docker image and run the command yourself inside of it.

Looking at this page seems to imply it is 3.22:

https://packages.ubuntu.com/search?keywords=cmake

Reply:
looks like its 3.25.2. ill update that, thanks. 

https://github.com/LLNL/uberenv/actions/runs/4108316351/jobs/7088871940#step:2:127